carma_cooperative_perception/include/carma_cooperative_perception/units_extensions.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef CARMA_COOPERATIVE_PERCEPTION_UNITS_EXTENSIONS_HPP_
+#define CARMA_COOPERATIVE_PERCEPTION_UNITS_EXTENSIONS_HPP_
+
+/*
+ * Copyright 2023 Leidos
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <units.h>
+
+#include <ratio>
+
+namespace carma_cooperative_perception
+{
+
+template <typename T>
+constexpr auto remove_units(const T & value) noexcept
+{
+  return units::unit_cast<typename T::underlying_type>(value);
+}
+
+}  // namespace carma_cooperative_perception
+
+namespace units
+{
+
+UNIT_ADD(
```

Comment:
Is this how we define new units that can be used? Can we add a comment mentioning this?

Reply:
Yes, I will like to the project's relevant documentation.